packages/connect/src/types/settings.ts

```diff
@@ -13,27 +13,20 @@ export interface ConnectSettingsPublic {
     manifest?: Manifest;
     connectSrc?: string;
     debug?: boolean;
-    hostLabel?: string;
-    hostIcon?: string;
     popup?: boolean;
     transportReconnect?: boolean;
-    webusb?: boolean; // deprecated
+    /** @deprecated */
+    webusb?: boolean;
```

Comment:
are we ready to remove it from types? (backwards comp stuff)?

Reply:
We can remove it, it looks like it doesn't even work anymore